packages/sit-onyx/src/components/OnyxListbox/OnyxMiniSearch.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { computed, useAttrs, type HtmlHTMLAttributes } from "vue";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import xSmall from "@sit-onyx/icons/x-small.svg?raw";
+import { injectI18n } from "../../i18n";
+
+defineOptions({ inheritAttrs: false });
+
+const attrs = useAttrs();
+
+const props = defineProps<{ modelValue: string; label: string }>();
+
+const emit = defineEmits<{
+  "update:modelValue": [input: string];
+}>();
+
+const { t } = injectI18n();
+
+/**
+ * Current value (with getter and setter) that can be used as "v-model" for the native input.
+ */
+const value = computed({
+  get: () => props.modelValue,
+  set: (value) => emit("update:modelValue", value),
+});
+
+const rootAttrs = computed(
+  () =>
+    ({ class: attrs["class"], style: attrs["style"] }) as Pick<
+      HtmlHTMLAttributes,
+      "class" | "style"
+    >,
+);
+
+const inputAttrs = computed<Omit<HtmlHTMLAttributes, "class" | "style">>(() => {
+  const rest = { ...attrs };
+  delete rest.class;
+  delete rest.style;
+  return rest;
+});
+</script>
+<template>
+  <div class="onyx-mini-search" v-bind="rootAttrs">
+    <input
+      v-model="value"
+      :aria-label="props.label"
+      v-bind="inputAttrs"
+      class="onyx-mini-search__input"
+      placeholder="Search"
+      type="text"
+    />
+    <!-- We use `@mousedown.prevent` here to not lose the input focus when the button is clicked  -->
+    <button
```

Comment:
button should be hidden when the search value is empty, right?

Reply:
sounds good